default.nix

```
@@ -133,9 +133,7 @@ rec {
     set -e
     export PATH=${ci-env}/bin:$PATH
     export TESTS=${tests_src}/integration_tests
-    pytest -v -n 3 -m 'not ledger and not upgrade' --dist loadscope $TESTS
-    pytest -v -m upgrade $TESTS
-    pytest -v -m ledger $TESTS
+    $1 $TESTS
```

```suggestion
    CMD=${1:-"pytest -v"}
    $1 $TESTS
```

in case someone `run-integration-tests` directly.